packages/cardano-services/src/Projection/prepareTypeormProjection.ts

```diff
@@ -92,13 +92,18 @@ const createMapperOperators = (
   const withHandleMetadata = handlePolicyIds
     ? Mapper.withHandleMetadata({ policyIds: handlePolicyIds }, logger)
     : passthrough();
+  const withGovernanceActions = projectionNames.includes(ProjectionName.ProtocolParameters)
+    ? Mapper.withGovernanceActions()
+    : passthrough();
+
```

Comment:
This shouldn't be needed, as it's included by specifying it in `storeMapperDependencies`
```suggestion
```

Reply:
855a05716f2 should solve this